container.go

```diff
@@ -860,6 +860,31 @@ func (c *Client) PutContainerArchive(id string, opts PutContainerArchiveOptions)
 	})
 }
 
+// GetContainerArchiveOptions is the set of options that can be used when
+// downloading resources from a container.
+//
+// See https://goo.gl/KnZJDX for more details.
+type GetContainerArchiveOptions struct {
+	OutputStream io.Writer `json:"-" qs:"-"`
+	Path         string    `qs:"path"`
```

Comment:
There are other functions that have multiple parameters. Opts are used for actually optional things, so this is confusing. id definitely should not be in opts, but path I also think should not be, I would be confused by that as a user and I use go-dockerclient all the time.